packages/api-client/src/http/BackendError.ts

```diff
@@ -17,18 +17,20 @@
  *
  */
 
-import {BackendErrorLabel, StatusCode} from '../http/';
+import type {StatusCodes as HTTP_STATUS} from 'http-status-codes';
+
+import {BackendErrorLabel} from '../http/';
 import {SyntheticErrorLabel} from './BackendErrorLabel';
 
 export class BackendError extends Error {
-  code: StatusCode;
+  code?: HTTP_STATUS;
   label: BackendErrorLabel | SyntheticErrorLabel;
   message: string;
 
   constructor(
     message: string,
     label: BackendErrorLabel | SyntheticErrorLabel = SyntheticErrorLabel.UNKNOWN,
-    code: StatusCode = StatusCode.UNKNOWN,
```

Comment:
Only caveat: We can't use our custom `UNKNOWN` type anymore.

Reply:
I am okay with that! 👍